.github/workflows/cypress-atomic-master.yml

```diff
@@ -0,0 +1,42 @@
+name: Cypress tests Atomic
+on:
+  push:
+    branches:
+      - master
+    tags-ignore:
+      - '*'
+jobs:
+  cypress-run:
+    name: Cypress run
+    runs-on: ubuntu-latest
+    strategy:
+      fail-fast: false
+      matrix:
+        containers: [1, 2, 3]
+    env:
+      NODE_OPTIONS: '--max_old_space_size=4096'
+    steps:
+      - uses: actions/checkout@v2
+      - name: Setup & Build
+        run: |
+          npm ci
+          npx lerna bootstrap
+          npm run build:e2e
+      - uses: cypress-io/github-action@v2
+        name: Cypress tests
+        with:
+          working-directory: ./packages/atomic
+          parallel: true
+          record: true
+          group: 'Atomic test runs'
+          start: npm run start:prod
+          wait-on: 'http://localhost:3333'
+          install: false
+        env:
+          CYPRESS_RECORD_KEY: ${{ secrets.CYPRESS_RECORD_KEY }}
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+      - uses: actions/upload-artifact@v1
+        if: failure()
+        with:
+          name: cypress-screenshots
+          path: packages/atomic/cypress/screenshots
```

Comment:
Aren't we losing a quality gate on the release process?

Builds on the `master` branch are failing frequently, for good or bad reasons, but this prevents bumps.

If passing e2e is not a prerequisite to creating a deployment pipeline package, I can see any one of us unknowingly promoting a package containing a regression, because we did not cross-check with the e2e github action.

Reply:
> and potentially slower builds on PR because it is not possible to run e2e tests in parallel?

No, build will be faster on PR versus what we had in Jenkins.

> More stable builds on PR, but potentially slower? If so, what is the expected difference in time?

Same as above.